lib/util/crashlytics_handler.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/material.dart';
+
+class FireBaseCrashLytics {
+  FireBaseCrashLytics.initialize() {
+    Crashlytics.instance.enableInDevMode = true;
+    FlutterError.onError = Crashlytics.instance.recordFlutterError;//to record any flutter error
+  }
+}
```

Comment:
Add a trailing newline character here

Reply:
before the comment?

Reply:
No, after line number 9. Every text file should end with a newline character.